editor/animation_track_editor.cpp

```diff
@@ -5297,6 +5297,32 @@ void AnimationTrackEditor::_add_track(int p_type) {
 		return;
 	}
 	adding_track_type = p_type;
+	Vector<StringName> valid_types;
+	switch (adding_track_type) {
+		case Animation::TYPE_METHOD:
+		case Animation::TYPE_BEZIER:
+		case Animation::TYPE_VALUE: { // "Property" track
+		} break;
```

Comment:
`adding_track_type` is `int`, so you don't need to handle unused cases.

Reply:
I took out these cases, it seems to work fine. You mentioned it's redundant because we're using an int, are there types where I would need to handle unused cases in a switch?

Reply:
If you switch enum, there is a warning/error that ensures you handle all cases. It would be caught by CI.